frontend/catalyst/device/qjit_device.py

```diff
@@ -95,12 +96,35 @@
     "Toffoli",
     "GlobalPhase",
 ]
+
+RUNTIME_OBSERVABLES = [
+    "Identity",
+    "PauliX",
+    "PauliY",
+    "PauliZ",
+    "Hadamard",
+    "Hermitian",
+    "Hamiltonian",
+    "Projector",
+    "SparseHamiltonian",
+    "Exp",
+    "LinearCombination",
```

Comment:
These three are not supported I believe, judging by [this function](https://github.com/PennyLaneAI/catalyst/blob/e9c8eccdcedbeeb9b41acd17ddffd36148f61e5c/frontend/catalyst/jax_tracer.py#L667):
```suggestion
    "Exp",
```

Reply:
`LinearCombination` is a `Sum` and should be handled by the `Sum` check. I don't see `Exp` though, am I missing something?

Reply:
Oh if it just works as a sum great! You're right, it seems `Exp` needs to be excluded as well.

Having tests for all of these would also tell which are(n't) supported :)